modules/datadog.nix

```diff
@@ -0,0 +1,13 @@
+{ config, pkgs, lib, ... }:
+
+with (import ./../lib.nix);
+
+{
+  config = {
+    services.dd-agent = {
```

Comment:
The name `dd-agent` comes from here, right? It would be nice to use `datadog-agent` to avoid confusion in the future.